### PR TITLE
updated logged in homepage 

### PR DESCRIPTION
**Pull request by aionate0812**
creating a trip while logged in will now tie the trip to a user

### CODE REVIEW

File: src/services/backendCalls.js [hunk 1]
```diff
@@ -37,10 +42,10 @@ const buildBundle = (
   destination,
   departureDate,
   returnDate,
-  user_id = null
+  user_uid = null
 ) => {
   let trip_id = null;
-  return createTrip(destination, departureDate, returnDate, (user_id = null))
```
Comment (JoshuaKGoldberg) [inline src/services/backendCalls.js:43]: Oh yeah, now that you mention it, this looks like a bug that you fixed. Nifty!

File: src/containers/Suggestions/Suggestions.js [hunk 2]
```diff
@@ -37,6 +41,12 @@ export default withRouter(props => {
     if (!currCategory) setDisplayItems(items.clothing);
   }, [items, currCategory]);
 
+  useEffect(() => {
+    if (FirebaseUserAuth.user !== null) {
+      setUser(FirebaseUserAuth.user.uid);
+    }
+  }, [FirebaseUserAuth, user]);
```
Comment (JoshuaKGoldberg) [inline src/containers/Suggestions/Suggestions.js:48]: Why is `user` a dependency? _(I'm not familiar with this too much)_

**Review by JoshuaKGoldberg — APPROVED** [review]
LGTM, just a couple of questions :)

File: src/services/backendCalls.js [hunk 3]
```diff
@@ -3,7 +3,12 @@ import { randomTripNameGenerator, splitDestination } from "./suggestions";
 
 const BASE_URL = "http://localhost:5000";
```
Comment (pamelaabreu) [inline src/services/backendCalls.js]: You can use backendUrlConnect in the services file to replace for base_url

**Review by pamelaabreu — APPROVED** [review]

**Pull request merged**